src/concrete/ml/sklearn/base.py

```diff
@@ -1328,6 +1339,279 @@ def __init__(self, n_bits: Union[int, Dict[str, int]]):
 
         BaseEstimator.__init__(self)
 
+    # TODO: FIX EXACT PREDICTION WITH HIGH BIT WIDTH
+    # pylint: disable=too-many-locals,too-many-statements,too-many-branches
+    @classmethod
+    def from_sklearn_model(
```

Comment:
We should put that somewhere else. It manipulates onnx which we shouldn't do here

Reply:
tree_to_numpy would be good. Maybe you can even rename this file to something more generic.

Reply:
renaming the file would technically be a breaking change, it has already been included in the list btw : https://github.com/zama-ai/concrete-ml-internal/issues/4159

Reply:
This file is internal. Why would it break anything from the API POV?

Reply:
because it's not explicitly internal (we haven't put any underscored in front of it)

so while I agree that it won't be a great risk to rename it as I doubt some users are really using it, it's still a breaking change (hence the "technically" from my answers above !). And I don't see any good reasons to rename this in a urgent manner, so imo waiting for the next major release is fine

Reply:
Fine with me too to wait